src/app/core/services/api/fake-backend/connector-fake-impl/data/test-policies.ts

```diff
@@ -26,12 +26,12 @@ export namespace TestPolicies {
       constraint(
         policyLeftExpressions.policyEvaluationTime,
         'GEQ',
-        '2020-12-31T23:00:00.000Z',
+        '2020-11-30T23:00:00.000Z',
       ),
       constraint(
         policyLeftExpressions.policyEvaluationTime,
         'LT',
```

Comment:
Shouldn't this be `LEQ`?

Reply:
no, check the month segment. UTC is minus one day at 22:00 or 23:00 for our midnight depending on summer time / winter time